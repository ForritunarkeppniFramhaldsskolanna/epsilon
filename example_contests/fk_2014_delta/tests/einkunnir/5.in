12
QDZ973 7.5
ETH464 7.5
ETH258 3.6
ICY879 5.5
XMZ353 6.0
ETH112 4.4
YMF324 3.7
ICY939 1.1
OZB771 7.4
YMF728 4.0
ICY737 3.2
XMZ550 2.4
