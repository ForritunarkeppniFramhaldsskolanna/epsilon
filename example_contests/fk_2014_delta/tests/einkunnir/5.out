ETH 5.16667
ICY 3.26667
OZB 7.4
QDZ 7.5
XMZ 4.2
YMF 3.85
